src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisConfigService.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server.service;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.security.SecurityAnalysisFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Service;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@Service
+@PropertySource("classpath:application.yaml")
```

Comment:
why you specify here the PropertySource ? it isn't application.yaml by default ?

Reply:
I need it to run unit test otherwise, it will look for an application-test.yml and I don't want to duplicate the config for tests